fastflix/widgets/panels/status_panel.py

```diff
@@ -163,6 +207,23 @@ def closeEvent(self, event):
         self.hide()
 
 
+class ElapsedTimeTicker(QtCore.QThread):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self.parent = parent
+        self.running = True
+
+    def __del__(self):
+        self.wait()
+
+    def run(self):
+        while self.running:
+            self.parent.signal_elapsed_time.emit("tick")
```

Comment:
`self.parent.signal_elapsed_time.emit()` don't need to pass anything if we remove the type when we declare the signal

Reply:
Awesome! That's exactly what I wished for :grinning: Added in c1e86d2